bot_jobs/get_national_data.py

```diff
@@ -0,0 +1,32 @@
+import pandas as pd
+
+URL = "https://raw.githubusercontent.com/wiki/djay/covidthailand/combined.csv"
+USE_COL = ["Hospitalized", "Cases", "Deaths"]
+START_DATE = "2021-01-01"
+
+# Get national data from URL
+df = pd.read_csv(URL)
+
+# Select Column and rename
+df = df[["Date",] + USE_COL]
```

Comment:
Looks weird, but ok, I guess 😆. Why use the name `USE_COL` for three columns? (the three columns aren't all)

Reply:
🤡

Reply:
So there are many columns that can be used from the `combined.csv` file. The selected column that is in the list is the column that we need and convert it from float to int as well. At first I thought we are going to need many columns from the file and I will get tired of typing all of the column out on many lines so I did this instead. What do you think? Might look a bit weird. Is there any better way?

Reply:
Ok, understandable in this case. No worries then, this looks good for now.